pages/k8s/backups.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Backups | Canonical Distribution of Kubernetes&reg;"
+keywords: juju, backup, etcd
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: backups.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+
+# Backups
+
+The state of your Kubernetes cluster is kept in the etcd datastore. This page
+shows how to backup and restore the etcd included in the **Canonical
+Distribution of Kubernetes <sup>&reg;</sup>**.
+
+ Backing up application specific data, normally stored in a persistent volume, is not
+ currently supported by native Kubernetes. Various third party solutions are available -
+ please refer to their own documentation for details.
+
+ ## Creating an **etcd** snapshot
+
+ <div class="p-notification--warning"><p class="p-notification__response">
+ <span class="p-notification__status">Warning:</span>
+ Snapshots can only be restored on the **same version of etcd**.
+  </p></div>
+
+
+
+ **etcd** is a distributed key/value store. To create a snapshot, all that is required is to run
+ the `snapshot` action on one of the units running **etcd**:
+
+ ```bash
+ juju run-action etcd/0 snapshot --wait
```

Comment:
@evilnick as mentioned in an earlier comment, i think we should be explicit with the snapshot `keys-version` parameter, especially since the default action param is `v2`, yet the default charm deployed is `v3`:

```
juju run-action etcd/0 snapshot keys-version=[v2|v3] --wait
```